src/commands/scanner/run.ts

```diff
@@ -180,6 +196,9 @@ export default class Run extends ScannerCommand {
 			this.ux.log(messages.getMessage('output.filtersIgnoredCustom', []));
 		}
 
+		// if severty-for-error flag is used, we want to make sure the severities are normalized
+		if (this.flags['severity-for-error']) this.flags['normalize-severity'] = true;
```

Comment:
Should this logic happen before line 130 where the engines are run? Also, can we have a separate boolean to hold the evaluated value so that we don't modify the input flag value?

Reply:
it happens in the validate flags functions, but I moved it to just be above 130 instead and use a separate variable